content/en/logs/guide/increase-number-of-log-files-tailed.md

```diff
@@ -15,13 +15,13 @@ further_reading:
   text: "How to investigate a log parsing issue?"
 ---
 
-By default the Agent can tail up to 100 log files. This limit is set to avoid performances issue when wildcards are set on huge directories.
+By default the Agent can tail up to 500 log files. This limit is set to avoid performances issue when wildcards are set on huge directories.
```

Comment:
```suggestion
By default the Agent can tail up to 200 log files on Windows and MacOS, and 500 log files on other operating systems. This limit is set to avoid performances issue when wildcards are set on huge directories.
```

Reply:
According to this PR: https://github.com/DataDog/datadog-agent/pull/12680

Reply:
LGTM 👍